### PR TITLE
Add section about naming classes

### PR DESCRIPTION
**Pull request by AlexVanderbist**
There's a lot more to say about naming things; for now this just adds the list of things to suffix.

(VS Code has also removed some trailing whitespace as a bonus 🙈)

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you also add some examples?

File: content/code-style/laravel-php.md [hunk 1]
```diff
@@ -464,3 +465,45 @@ Translations must be rendered with the `__` function. We prefer using this over
...
+
+E.g. `CreateUser` or `PerformDatabaseCleanup`
+
+### Events
+
+Events will often be fired before or after the actual event. This should be very clear by the tense used in their name.
+
+E.g. `ApprovingLoan` before the action is completed and `LoanApproved` after the action is completed.
+
+### Listerens
```
Comment (sebastiandedeyne) [inline content/code-style/laravel-php.md]: Typo!

Comment (sebastiandedeyne) [reply content/code-style/laravel-php.md]: Maybe "Listeners & Subscribers"?

File: content/code-style/laravel-php.md [hunk 2]
```diff
@@ -464,3 +465,45 @@ Translations must be rendered with the `__` function. We prefer using this over
...
+
+Listeners will perform an action based on an incomming event. Their name should reflect that action with a `Listener` suffix. This might seem strange at first but will avoid naming collisions with jobs.
+
+E.g. `SendInvitationMailListener`
+
+### Mailables
```
Comment (sebastiandedeyne) [inline content/code-style/laravel-php.md]: Should we mention notifications too?

**Comment by sebastiandedeyne** [comment]
Looks good! Added a few comments. Should we discuss Artisan `Command` classes too?

File: content/code-style/laravel-php.md [hunk 3]
```diff
@@ -464,3 +465,45 @@ Translations must be rendered with the `__` function. We prefer using this over
...
+
+### Events
+
+Events will often be fired before or after the actual event. This should be very clear by the tense used in their name.
+
+E.g. `ApprovingLoan` before the action is completed and `LoanApproved` after the action is completed.
+
+### Listerens
+
+Listeners will perform an action based on an incomming event. Their name should reflect that action with a `Listener` suffix. This might seem strange at first but will avoid naming collisions with jobs.
```
Comment (freekmurze) [inline content/code-style/laravel-php.md]: Typo here: `incoming`

**Comment by freekmurze** [comment]
Let's discuss `Command` and `Notification` suffixes together at lunch.

**Pull request merged**